internal/js/modules/k6/browser/common/selectors.go

```diff
@@ -22,6 +22,12 @@ import (
 	"strings"
 )
 
+// TODO: Reimplement this by looking at:
+// https://github.com/microsoft/playwright/blob/
+// 		35cd6fb279ccf4c91eae4b8094697eceb61426cc/
+// 		packages/playwright-core/src/utils/isomorphic/selectorParser.ts#L154
+// Current implementation is a simplified version of the above and cannot handle all edge cases.
+
```

Comment:
I only think we should reimplement if there is a good use case for it.

I feel that in the browser module lots of things were implemented, but only now we're understanding and using changes made from the initial commit.

Once we understand the use case, then reimplementation will be easier, otherwise we will be doing a lot of work without understanding the problem.

Reply:
👍 Fair enough. Let's track this in a new issue, but can we remove the TODO comment?

Reply:
I'll create an issue and change the TODO to a link to that issue 👍